EnergyCorrelatorOptions.h

```diff
@@ -14,9 +14,9 @@
 #include <utility>
 #include <optional>
 // analysis utilities
-#include "/sphenix/user/danderson/install/include/scorrelatorutilities/Types.h"
-#include "/sphenix/user/danderson/install/include/scorrelatorutilities/Constants.h"
-#include "/sphenix/user/danderson/install/include/senergycorrelator/SEnergyCorrelatorConfig.h"
+#include "/sphenix/user/atclarke/install/include/scorrelatorutilities/Types.h"
+#include "/sphenix/user/atclarke/install/include/scorrelatorutilities/Constants.h"
+#include "/sphenix/user/atclarke/install/include/senergycorrelator/SEnergyCorrelatorConfig.h"
```

Comment:
The reason why the full path is needed here is because I haven't bothered to add my install directory to my $LD_LIBRARY path. Now that we're juggling installs, I'll finally do that...

(And I'll let you know how to do that on your end if you haven't added it as well!)

```suggestion
#include "scorrelatorutilities/Types.h"
#include "scorrelatorutilities/Constants.h"
#include "senergycorrelator/SEnergyCorrelatorConfig.h"
```